crates/wasmtime/src/runtime/module.rs

```diff
@@ -242,7 +243,7 @@ impl Module {
         let bytes = bytes.as_ref();
         #[cfg(feature = "wat")]
         let bytes = wat::parse_bytes(bytes)?;
-        Self::from_binary(engine, &bytes)
+        Self::from_binary(engine, &bytes, None)
```

Comment:
If you're ok changing these constructors of `Module` I think it'd be best to reframe them all in terms of `ModuleBuilder`, but I can also have a follow-up to do that refactoring.